### ISSUE TITLE
Enforce/suggest an LVCSR task creation template

### ISSUE THREAD

**Issue by vijayaditya** [kaldi-asr/kaldi#3385]
I have come across few tasks (e.g. `egs/tedlium/`) which do not provide a README. This makes it difficult for experimenters to start working on this task. It would be great if we start adding the necessary details in a README for the existing tasks and create an LVCSR task template directory for new tasks to aid reviewers/task-designers.
e.g. 
```bash
egs/template/
|----->README.md (details of the task, link to the paper if available)
|----->s5/README.md (details of the recipe, compare and contrast with other recipes in this task if necessary)
```

This issue is created to discuss the template and track any PRs adding READMEs for the existing tasks.

**Comment by danpovey** [comment]
Makes sense.  I'd probably rather just to keep them plain text files (README), since it's always been like that and it will avoid the question of whether to upgrade the old ones.  But open to opinions on that.

**Comment by vijayaditya** [comment]
I didn't see any complex markdown features being used in the existing README.md, so the plain text files might be sufficient.
